IgOmeProfiling_pipeline.py

```diff
@@ -166,6 +166,8 @@ def run_pipeline(fastq_path, barcode2samplename_path, samplename2biologicalcondi
     parser.add_argument('--tfidf_method', choices=['boolean', 'terms', 'log', 'augmented'], default='boolean', help='TF-IDF method')
     parser.add_argument('--tfidf_factor', type=float, default=0.5, help='TF-IDF augmented method factor (0-1)')
     parser.add_argument('--shuffles', default=5, type=int, help='Number of controlled shuffles permutations')
+    parser.add_argument('--percent', default=0.2, type=float, help='Percent from the bigit hit shuffle')
```

Comment:
change flag name to `shuffles_percent`.
Change in description `the bigit hit shuffle` to `shuffle with greatest number of hits (0-1)`